docs/ANDROID.md

```diff
@@ -1,13 +1,36 @@
 # Android support
 
-## Building Hermes from source
+## Android setup
+
+### Step 1: `settings.gradle`
+
+Gradle needs special handling to build React Native from source. In your app's `settings.gradle` please include the below:
+
+```groovy
+// customize the path to match your node_modules location
+apply(from: "../../../node_modules/react-native-node-api/android/consumerSettings.gradle")
+applyNodeAPISettings(settings)
+```
+
+### Step 2: script for adjusting environment variables
+
+To integrate automatic setup of Hermes engine, a special env variable (`REACT_NATIVE_OVERRIDE_HERMES_DIR`) must be set to a proper path. Since Gradle does not really support loading `.env` files, this must be automated by the consumer. We provide the script `react-native-node-api vendor-hermes --silent` which will output a single line, the path to Hermes directory.
```

Comment:
```suggestion
A special environment variable (`REACT_NATIVE_OVERRIDE_HERMES_DIR`) must be set to the path of a Hermes engine with Node-API support. Since Gradle does not support loading `.env` files directly, this must be automated by the consumer. We provide the `react-native-node-api vendor-hermes --silent` command, which will download Hermes and output the path to Hermes directory path as its only output.
```

Reply:
Thanks, this is more clear - applied.